content/en/docs/pipelines/installation/overview.md

```diff
@@ -75,6 +73,13 @@ Upgrade Support (**Beta**)
 : [Upgrading Kubeflow Pipelines Standalone](/docs/pipelines/installation/standalone-deployment/#upgrading-kubeflow-pipelines) introduces how to upgrade
 in place.
 
+Google Cloud Integrations
+:
+  * A Kubeflow Pipelines public endpoint with auth support is **auto-configured** for you.
+  * Open Kubeflow Pipelines UI via the **Open Pipelines Dashboard** link in [the AI Platform Pipelines dashboard of Google Cloud Console](https://console.cloud.google.com/ai-platform/pipelines/clusters).
+  * (Optional) You can choose to persist your data in Google Cloud managed storages (Cloud SQL and Cloud Storage).
```

Comment:
Minor proposal:
- Hyphenate

```diff
- * ... your data in Google Cloud managed storages
+ * ... your data in Google Cloud-managed storages
```

Reply:
I'm not sure about grammer, but it's "Google Cloud" and "managed storages", so far we are consistently using this phrase.

Reply:
I would recommend against hyphenating that. 

Also,
storages -> storage